### PR TITLE
chore(IDX): container user

### PR DESCRIPTION
**Pull request by marko-k0**
Rebuilding `ic-build` image and setting `IC_USER=1001` as `ARG`. Follow symlinks with `chown -RL /cache`.

### CODE REVIEW

File: bazel/external_crates.bzl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-This module declares all direct rust dependencies.
+This module declares all direct rust dependencies!
```
Comment (basvandijk) [inline bazel/external_crates.bzl:2]: Was this change intended?

**Review by cgundy — APPROVED** [review]

**Review by basvandijk — APPROVED** [review]

File: bazel/external_crates.bzl (hunk 1 above)
Comment (marko-k0) [reply bazel/external_crates.bzl:2]: No, I'll put it out as soon as current workflow run is done.
